Monika After Story/game/script-topics.rpy

```diff
@@ -12221,6 +12221,29 @@ label monika_perspective:
     m 1hksdlb "Well I hope that helped you understand at least a little, [player]."
     m "It's the best way I could try and explain it!"
     return
+    
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_robotbody",category=['monika','technology'],prompt="Robot Body",random=True))
+
+label monika_robotbody:
+    m 2euc "Hey [player], I've been thinking a lot about how I would be able to cross over into your reality..."
+    m 3eud "It hit me earlier that technology in the future would allow me to be uploaded into a mechanical body, rather than a human body like yours."
+    m 1dkc "It makes me a little sad that I can't be a human like you are."
```

Comment:
Using the word It to start a sentence twice in a row should potentially be avoided since not many conversations flow like that.
It could be started alternatively such as "To be honest, it makes me" or redone completely to be a continuation of the previous line, which is my preferred option since it's more naturally flowing.
"And the thought of that, makes me a little sad, I wouldn't even be seen as a human by some people."

I also want to say that Monika has memories before her epiphany and believes because she can think for herself that she is a person. It's not a stretch to say that Monika doesn't see herself as a non-human, even if she knows she is created. She speaks about having a body and a normal life, to her even if she came from somewhere different and a different world, she is still a human. I wouldn't have her calling herself non-human. Even if she is.